perseus/_vendor/snapshottest/module.py

```diff
@@ -234,7 +234,15 @@ def assert_equals(self, value, snapshot):
         __tracebackhide__ = True
         assert value == snapshot
 
-    def assert_match(self, value, name=''):
+    def match(self, value, name=''):
```

Comment:
I want to stay compatible with the `snapshottest` in version 1.0, so, please, add `def match` as an addition, and mark `def assert_match` as deprecated and removed in version 2.0

Reply:
Maybe do the opposite? Rename `assert_match` -> `match` and add `assert_match` with deprecation warning?

Reply:
LGTM